site/en/blog/chrome-109-beta/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Chrome 109 beta
+description: >
+  MathML, conditional focus for getDisplayMedia(), Origin Private File System on Android, and more.
+subhead: >
+  MathML, conditional focus for getDisplayMedia(), Origin Private File System on Android, and more.
+layout: 'layouts/blog-post.njk'
+date: 2022-12-01
+hero: 'image/kheDArv5csY6rvQUJDbWRscckLr1/zQuGajDm8rdY6kUo84LB.png'
+alt: >
+  Chrome 109 beta hero logo
+tags:
+  - beta
+  - chrome-109
+---
+
+Unless otherwise noted, changes described below apply to the newest Chrome beta channel release for Android, ChromeOS, Linux, macOS, and Windows. Learn more about the features listed here through the provided links or from the list on ChromeStatus.com. Chrome 109 is beta as of December 1, 2022. You can download the latest on [Google.com](https://www.google.com/chrome/beta/) for desktop or on Google Play Store on Android.
+
+## CSS
+
+### Auto range support for font descriptors inside `@font-face` rule
+
+Variable fonts provide users the opportunity to choose how heavy or slanted or wide the typeface should be, using the `font-weight`, `font-style`, and `font-stretch`' descriptors inside the `@font-face` rule. The CSS Working Group added a new value of `auto` for these descriptors, which is now the initial value. This value is implemented in 109 to match the new specification.
+
+### CSS `lh` Length Unit
+
+The `lh` CSS [`<length>`](https://developer.mozilla.org/docs/Web/CSS/length) unit is equivalent to the computed value of the line-height property on the element on which it is used. This allows a `<textarea>` to be given a height equivalent to the number of lines of text expected.
+
+### CSS `hyphenate-limit-chars` property
+
+The `hyphenate-limit-chars` property specifies the minimum number of characters in a hyphenated word. When applying the hyphenation, the optimal minimum number of characters in the word, before the hyphen, or after the hyphen can vary by the design of the page, or by the language. This property allows finer grained control of the hyphenation for better typography on the web. It can also help international pages when the default settings are not optimal. 
+
+### Snap border, outline and column-rule widths before layout
+
+Currently Blink snaps the border widths at paint time. This can cause a visible 1px gap between a parent element's border and a child's background when the border is rounded down during paint, since the snapping floors the value but the layout rounded it up to calculate the child’s position (for example, border-width is set to 10.75 pixels, rounded to 10px at paint time, but 11 at layout time). By making this change Blink will behave like Gecko and WebKit, improving interoperability.
```

Comment:
```
This can cause a visible 1px gap between a parent element's border and a child's background when the border is rounded down during paint, since the snapping floors the value but the layout rounded it up to calculate the child’s position (for example, border-width is set to 10.75 pixels, rounded to 10px at paint time, but 11 at layout time).
```

I'm not really clear on what this means. I think it could do with reworking into 2 sentences.

Reply:
@rachelandrew Just one comment. I committed a couple of tiny things